src/services/organizationService.ts

```diff
@@ -351,10 +453,11 @@ export async function registerOrganization(
 }
 
 export async function getOrganizationInstanceFromDelegateEmail(
-  userEmail: string
+  userEmail: string,
+  ipaCode?: string
 ): Promise<Either<Error, Option<OrganizationModel>>> {
   try {
-    const organizationInstance = await OrganizationModel.findOne({
+    const organizationInstances = await OrganizationModel.findAll({
```

Comment:
can we do a `count()` instead ?

Reply:
It wouldn't be enough. This method should return the organization to which the user is associated as a delegate, so it's anyway needed to query the database in order to get the organization.